CSS/css_completions.py

```diff
@@ -426,7 +426,7 @@ class CSSCompletions(sublime_plugin.EventListener):
     regex = None
 
     def on_query_completions(self, view, prefix, locations):
-        selector_scope = "source.css - meta.selector.css"
+        selector_scope = "source.css - meta.selector.css, text.html meta.attribute-with-value.style.html string.quoted punctuation.definition.string.end.html"
```

Comment:
Let's imagine we have the following HTML:

    <div style="float: "></div>

and the caret is immediately before the closing `"` character. Using the `view.scope_name(point)` API / `view.match_selector(point, selector)` API (where the `point` is the beginning of the selection region) will return the scope of the `"` character, i.e. `punctuation.definition.string.end.html`

If the caret position is immediately after the `"` character, it would return the scope of the next character to the right, in this example case a `>`, so `punctuation.definition.tag.end.html`.

Therefore, as we want completion of CSS to still occur while the next character is the closing quote, we use the scope selector `punctuation.definition.string.end.html`, which will trigger just before the closing quote of the attribute, and not just after it :)

Probably to really see the difference, I suggest trying it out, both with and without this change. :)